pkgs/development/libraries/libguestfs/default.nix

```diff
@@ -12,19 +12,19 @@ assert javaSupport -> jdk != null;
 
 stdenv.mkDerivation rec {
   pname = "libguestfs";
-  version = "1.40.2";
+  version = "1.44.1";
 
   src = fetchurl {
-    url = "https://libguestfs.org/download/1.40-stable/${pname}-${version}.tar.gz";
-    sha256 = "ad6562c48c38e922a314cb45a90996843d81045595c4917f66b02a6c2dfe8058";
+    url = "https://libguestfs.org/download/1.44-stable/${pname}-${version}.tar.gz";
+    sha256 = "09dhmlbfdwirlmkasa28x69vqs5xndq0lnng6b4if76s6bfxrdvj";
   };
 
-  nativeBuildInputs = [ autoreconfHook makeWrapper pkg-config ];
+  nativeBuildInputs = [ autoreconfHook bison flex getopt makeWrapper pkg-config ];
```

Comment:
Can we set strictDeps?

Reply:
I just enabled strictDeps and moved cdrkit, cpio, gperf, OCaml, Perl to nativeBuildInputs to make it build. I fear some other dependencies in `buildInputs` should go in `nativeBuildInputs`.